.github/workflows/sonar.yml

```
@@ -0,0 +1,40 @@
+    branches:
+      - master
+      - main
+      - release-*
+
+jobs:
+  build:
+    name: Sonarqube_CI
+    if: ${{ github.event.label.name == 'sonar_check' || github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.10)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)